client/src/app/layout.tsx

```diff
@@ -25,6 +26,7 @@ export default function RootLayout({
         <ThemeProvider attribute="class" enableSystem>
           <div className="relative max-w-6xl mx-auto">
             <Navbar />
+            <Toaster position="bottom-left"/>
```

Comment:
Change to top center

Reply:
Ok, but I am interested to know why it not should be in global....?

Reply:
So if we add this here it will trigger for every component as layout is global and it should render on every component even we that component is not using this toast notification. Only add on layout if you need globally like for state management, dark/light mode, etc.

I guess you got it now.

Reply:
Yep got it 😀 👍🏻